content/en/docs/concepts/security/_index.md

```diff
@@ -3,4 +3,241 @@ title: "Security"
+encryption keys. If you use Kubernetes' own API for
+[CertificateSigningRequests](/docs/reference/access-authn-authz/certificate-signing-requests/#certificate-signing-requests),
+pay special attention to restricting misuse there.
+
+### Runtime protection: compute {#protection-runtime-compute}
+
+{{< glossary_tooltip text="Containers" term_id="container" >}} provide two
+things: isolation between different applications, and a mechanism to combine
+those isolated applications to run on the same host computer. Those two
+aspects, isolation and aggregation, mean that runtime security involves
+trade-offs and finding an appropriate balance.
+
+Kubernetes relies on a {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+to actually set up and run containers. The Kubernetes project does
+not recommend a specific container runtime and you should make sure that
+the runtime(s) that you choose meet your information security needs.
+
+To protect your compute at runtime, you can:
+
+1. Enforce [Pod security standards](/docs/concepts/security/pod-security-standards/)
+   for applications, based on the level of trust you have in them.
+1. Run a specialized operating system on your nodes that is designed specifically
+   for running containerized workloads. This is typically based on a read-only
+   operating system (_immutable image_) that provides only the services
+   essential for running containers.
+
+   Container-specific operating systems help to isolate system components and
+   present a reduced attack surface in case of a container escape.
+1. Define [ResourceQuotas](/docs/concepts/policy/resource-quotas/) to
+   fairly allocate shared resources, and use
+   mechanisms such as [LimitRanges](/docs/concepts/policy/limit-range/)
+   to ensure that Pods specify their resource requirements.
+1. Partition workloads across different nodes.
+   Use [node isolation](/docs/concepts/scheduling-eviction/assign-pod-node/#node-isolation-restriction)
+   mechanisms, either from Kubernetes itself or from the ecosystem, to ensure that
+   Pods with different trust contexts are run on separate sets of nodes.
+1. Use a {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+   that provides security restrictions.
+1. On Linux nodes, use a Linux security module such as [AppArmor](/docs/tutorials/security/apparmor/) (beta)
+   or [seccomp](/docs/tutorials/security/seccomp/).
+
+### Runtime protection: storage {#protection-runtime-storage}
+
+To protect storage for your cluster and the applications that run there, you can:
+
+1. Integrate your cluster with an external storage plugin that provides encryption at
+   rest for volumes.
+1. Enable [encryption at rest](/docs/tasks/administer-cluster/encrypt-data/) for
+   API objects.
+1. Protect data durability using backups. Verify you can restore these if you need to.
+1. Authenticate connections between cluster nodes and any network storage they rely
+   upon.
```

Comment:
```suggestion
   upon it.
```

Reply:
I'm afraid I won't accept this suggestion as it doesn't look like typical English grammar.